starter_code/app.js

```diff
@@ -1,18 +1,51 @@
+'use strict';
 
 const express = require('express');
-const hbs     = require('hbs');
-const app     = express();
-const path    = require('path');
+const hbs = require('hbs');
+const app = express();
+const path = require('path');
+
 const PunkAPIWrapper = require('punkapi-javascript-wrapper');
 const punkAPI = new PunkAPIWrapper();
 
+// ------- configure app
+
+app.set('views', path.join(__dirname, '/views'));
+
 app.set('view engine', 'hbs');
-app.set('views', __dirname + '/views');
 app.use(express.static(path.join(__dirname, 'public')));
+hbs.registerPartials(path.join(__dirname, '/views/partials'));
+
+// ------ routes
 
 app.get('/', (req, res, next) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res, next) => {
+  punkAPI.getBeers()
+    .then(beersResults => {
+      // console.log(beersResults[0]);
+      const data = {beersResults};
+      res.render('beers', data);
+    })
+    .catch(error => {
+      console.log(error);
+    });
+});
+
+app.get('/randombeer', (req, res, next) => {
+  punkAPI.getRandom()
+    .then(randomBeer => {
+      // console.log(randomBeer[0])
+      const randomData = { randomBeer: randomBeer[0] }; // Take some time to understand this more
```

Comment:
just ```const data``` is ok, cause you can pass more stuff there and maybe this other stuff is not random, so you are passing ```data``` and inside it there's a randomBeer